tmt/steps/report/reportportal.py

```diff
@@ -0,0 +1,62 @@
+import io
+import os
+import zipfile
+
+import junit_xml
+import requests
+
+import tmt.steps.report
+import tmt.utils
+
+from . import junit
+
+
+class ReportPortal(tmt.steps.report.ReportPlugin):
+    """
+    Send results in JUnit format to ReportPortal instance.
+
+    Requires environment variables TMT_REPORT_PORTAL_URL and TMT_REPORT_PORTAL_TOKEN.
+    """
+
+    _methods = [tmt.steps.Method(name="reportportal", doc=__doc__, order=60)]
```

Comment:
Use `provides_method` decorator instead of `_methods` (e.g. https://github.com/teemtee/tmt/blob/main/tmt/steps/report/junit.py#L94) - `_methods` is deprecated, and it should raise a warning or two in runtime.